Assets/Production/Prefabs/Enemy 1.prefab

```diff
@@ -62,7 +62,7 @@ MonoBehaviour:
   memoryDuration: 2
   lastSeenTime: 0
   lastSeenLocation: {x: 0, y: 0, z: 0}
-  attackDamage: 25
+  attackDamage: 1
```

Comment:
should this be reverted

Reply:
nope :), thats correct
Enemies should do only do 1 hp damage per hit

Reply:
awesome, just double cchecking